lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -366,32 +364,22 @@ def start_watch
       watcher.each do |notice|
         case notice.type
           when 'MODIFIED'
-            if notice.object.status.containerStatuses
```

Comment:
Ok - yeah, if the field is always set it doesn't make much sense to have a condition for it.